README.md

```diff
@@ -0,0 +1,172 @@
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+
+If you dont have Ruby installed on your computer, you can download it from [here](https://www.ruby-lang.org/en/downloads/).
+
+If you dont have Rails installed on your computer, you can download it from [here](https://rubyonrails.org/).
+
+If you dont have PostgreSQL installed on your computer, you can download it from [here](https://www.postgresql.org/download/).
+
+If you have installed git you can clone the code to your machine, or download a ZIP of all the files directly.
+
+
+> `NOTE:` _You may need to run the following commands in the project directory to install the required gems and run the application:_
+
+1. Install gem packages with:
+
+```
+bundle install
+```
+
+2. Open the config/database.yml file in the project directory and change the username and password to your PostgreSQL username and password.
+Edit the default section of the file to look like this:
+
+```
+  default: &default
+  adapter: postgresql
+  encoding: unicode
+  # For details on connection pooling, see Rails configuration guide
+  # http://guides.rubyonrails.org/configuring.html#database-pooling
+  pool: <%= ENV.fetch("RAILS_MAX_THREADS") { 5 } %>
+  username: <your PostgreSQL role username>
+  password: <your PostgreSQL role password>
+
+```
+
+3. Create the database with:
+
+```
+rails db:create
+```
+
+4. Start the development server
+
+```
+rails server
+```
+
+5. Open the app in your browser at http://localhost:3000
+### Run tests
+
+To run tests, run the following command:
+rspec
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Riya Bulia**
+
+- GitHub: [@RiyaBulia12](https://github.com/RiyaBulia12)
+- LinkedIn: [Riya-Bulia](https://linkedin.com/in/riya-bulia)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- [Edit and Delete Action on Category] **[Edit and Delete Action on Category]**
+- [Edit and Delete Action on Transaction] **[Edit and Delete Action on Transaction]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project give as a star! ⭐️
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+- Hat tip to anyone whose code was used
+- [Microverse Document](https://github.com/microverseinc/curriculum-rails/blob/main/capstone/rails_capstone.md)
```

Comment:
- [ ] Please give appropriate credit to [Gregoire Vella on Behance](https://www.behance.net/gregoirevella) the [author of the original design](https://www.behance.net/gallery/19759151/Snapscan-iOs-design-and-branding?tracking_source=), as required by the [Creative Commons license of the design](https://creativecommons.org/licenses/by-nc/4.0/).